### ISSUE TITLE
Remove WPTs for spec-removed `navigate-to` directive

### ISSUE THREAD

**Issue by CanadaHonk** [w3c/webappsec-csp#608]
The `navigate-to` directive was removed in September 2022 (https://github.com/w3c/webappsec-csp/pull/564) but WPTs still exist. They should likely be simplified into one historical test ensuring it is *not* supported.

See also: [Gecko patch removing impl with WPT discussion](https://phabricator.services.mozilla.com/D181630)

**Comment by CanadaHonk** [comment]
Fwiw, I already created a historical test for the patch but it was backed out as other vendor decisions hadn't been made fully sure of yet. Not sure if it should be confirmed formally (here or otherwise) first, or if a PR to WPT can just be created?

**Comment by annevk** [comment]
I don't think we need that. `historical.*` (with its current de facto semantics of testing the negative) is good enough and makes it clear the feature is proprietary and not to be implemented. Those tests are also a lot simpler often than actually testing support as you don't need to care about edge cases as much. If browsers nevertheless want to maintain support I suppose they can keep around `.tentative` tests although hopefully not for too long.

@CanadaHonk it's nice to confirm via an issue like this, but it's not required as things stand today. Ideally this would have already been done as part of removing the feature, but not all groups keep track of their tests as well as they should.
